BusBoard.ConsoleApp/TflApi.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Newtonsoft.Json;
+using RestSharp;
+
+namespace BusBoard.ConsoleApp
+{
+    public class TflApi
+    {
+        public IEnumerable<BusStop> GetTwoClosestBusStopsToPostcode(string postcodeString)
+        {
+            var requestUrl = "http://api.postcodes.io";
```

Comment:
I think this logic should live in its own PostcodeApi class. You could call it from your main method and then pass the values in to on of your TfL methods from there.

This is about separation of concerns and making sure each class is only handling one thing.

Reply:
Ah yeah of course, forgot they were different Apis